### PR TITLE
chore: add p3 label

### PR DESCRIPTION
**Pull request by kolea2**
I'm looking for suggestions for the description, and if we need to update anything else. PTAL :)

### CODE REVIEW

File: packages/label-sync/src/labels.json [hunk 1]
```diff
@@ -55,6 +55,11 @@
       "description": "Request for information or clarification. Not an issue.",
       "color": "c5def5"
     },
+    {
+      "name": "priority: p3",
+      "description": "",
```
Comment (crwilcox) [inline packages/label-sync/src/labels.json]: I may suggest that we remove the 'highest, medium, lowest' from others, and update descriptions a bit. For this one, perhaps "Desirable enhancement or fix. May not be include in next release"

File: packages/label-sync/src/labels.json [hunk 2]
```diff
@@ -55,6 +55,11 @@
...
+      "color": "ffffc7"
+    },
     {
       "name": "priority: p2",
       "description": "Lowest priority. Fix may not be included in next release.",
```
Comment (crwilcox) [inline packages/label-sync/src/labels.json]: P2 Current: Lowest priority. Fix may not be included in next release.
P2 Suggest: Moderately-important Issue. Fix may not be included in next release.

P1 Current: Medium priority. Will be fixed prior to next release.
P1 Suggest: Important issue which blocks shipping the next release. Will be fixed prior to next release.

P0 Current: Highest priority. Critical issue. P0 implies highest priority.
P0 Suggest: -

**Comment by kolea2** [comment]
Thanks @crwilcox! Updated based on your suggestions.

File: packages/label-sync/src/labels.json [hunk 3]
```diff
@@ -55,14 +55,19 @@
       "description": "Request for information or clarification. Not an issue.",
       "color": "c5def5"
     },
+    {
+      "name": "priority: p3",
+      "description": "Desirable enhancement or fix. May not be include in next release.",
```
Comment (crwilcox) [inline packages/label-sync/src/labels.json]:
```suggestion
      "description": "Desirable enhancement or fix. May not be included in next release.",
```

Comment (kolea2) [reply packages/label-sync/src/labels.json]: oops sorry, missed this. I'll create a typo fix

Comment (crwilcox) [reply packages/label-sync/src/labels.json]: sgtm. thanks!

**Pull request merged**